code/modules/uplink/uplink_item.dm

```diff
@@ -299,6 +299,20 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	item = /obj/item/melee/powerfist
 	cost = 8
 
+/datum/uplink_item/dangerous/mindbatterer
+	name = "Psychic Disruptor"
+	desc = "A new Epsilon Psi Corps Disruptor Device, also known as Mind Batterer. Creates an invisible wave \
+		of psychic energy that causes immense pain to people near the user \
+		forcing them to the ground. People can resist the psychic blast. \
```

Comment:
`People can resist the psychic blast.` why do they fall to the ground then, what.

Not a fan of making it a limited stock item, since it's not intuitive and there isn't a lot that tells people about the limitedness of things.

Reply:
Dunno what I was smoking last night about the resist thing. Should rephrase it to something like "It attempts to overwhelm people with an incredibly painfull headache"

Also reverting the limited stock thing considering I put it into place for the pre nerfed version with a knockdown which lasted 10-20 seconds which was much much more potent.